### PR TITLE
GLUE-5470. Add glue specific constants rule.

### PR DESCRIPTION
**Pull request by ghost**
Ticket: https://spryker.atlassian.net/browse/GLUE-5470

Related PRs:
* https://github.com/spryker/git-hook/pull/29
* https://github.com/spryker/suite-nonsplit/pull/1855

### CODE REVIEW

File: GlueStreamSpecific/Sniffs/Classes/StringOnlyInConstantSniff.php [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+    /**
+     * @inheritdoc
+     */
+    public function process(File $phpcsFile, $stackPtr)
+    {
+        $decName = $phpcsFile->findNext(T_CONSTANT_ENCAPSED_STRING, $stackPtr);
+        $tokens = $phpcsFile->getTokens();
+
+        if ($tokens[$decName]['level'] === 1) {
```
Comment (dereuromark) [inline GlueStreamSpecific/Sniffs/Classes/StringOnlyInConstantSniff.php]: You need to code more defensive
$decNameIndex could not be set here:  if (!$decNameIndex) return etc.

File: GlueStreamSpecific/ruleset.xml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<ruleset name="Glue specific ruleset">
+    <description>
+        Glue specific Coding Standard(experimental).
```
Comment (dereuromark) [inline GlueStreamSpecific/ruleset.xml]: space missing before (

File: GlueStreamSpecific/Sniffs/Classes/StringOnlyInConstantSniff.php (hunk 1 above)
Comment (sergeysikachev) [reply GlueStreamSpecific/Sniffs/Classes/StringOnlyInConstantSniff.php]: Yeah, didn't notice bool in possible return types

**Comment by sergeysikachev** [comment]
@dereuromark , did you check related PRs? Especially this one, where a possibility of stream specific quality gate is introduced spryker/git-hook#29

**Pull request merged**